.vscode/settings.json

```diff
@@ -44,16 +44,356 @@
         }
     ],
     "latex-workshop.viewer.pdf.internal.port": 41491,
-    "spellright.documentTypes": [
-        "markdown",
-        "latex",
-        "plaintext"
+    "spellright.documentTypes": [ ],
```

Comment:
missed one

Reply:
This one is left on purpose to avoid having to manually disable the spellright extension if it's installed. I couldn't find a better way to disable it via the config files.